ui/src/app/components/app-router.tsx

```diff
@@ -145,17 +99,12 @@ export const AppRoute = ({
     fp.find(({ allowed }) => !allowed(), guards) || {};
 
   return (
-    <Route exact={exact} path={path}>
```

Comment:
changing Route to CompatRoute is part of the migration path.
CompatRoute has *both* v5 and v6 apis, so we can convert at whatever pace is appropriate, then convert to a v6 Route when done